src/test/java/com/glencoesoftware/omero/zarr/ZarrPixelBufferTest.java

```diff
@@ -338,11 +326,8 @@ public void testGetTile() throws IOException, InvalidRangeException {
         int sizeZ = 4;
         int sizeY = 5;
         int sizeX = 6;
-        int resolutions = 1;
-        Pixels pixels = new Pixels(
-                null, null, sizeX, sizeY, sizeZ, sizeC, sizeT, "", null);
-        Path output = writeTestZarr(
-                sizeT, sizeC, sizeZ, sizeY, sizeX, "int32", resolutions);
+        Pixels pixels = new Pixels(null, null, sizeX, sizeY, sizeZ, sizeC, sizeT, "", null);
+        Path output = writeTestZarr(sizeT, sizeC, sizeZ, sizeY, sizeX, "int32");
         ZarrArray test = ZarrArray.open(output.resolve("0").resolve("0"));
         int[] data = new int[2 * 3 * 4 * 5 * 6];
```

Comment:
Not introduced by this PR, but here and elsewhere the size variables are ignored in favor of hard-coded dimensions. Doesn't necessarily need to be changed in this PR, but probably worth updating at some point.